libs/types/src/cast_vote_record.ts

```diff
@@ -18,12 +18,12 @@ export interface CastVoteRecord
     | number
     | readonly number[]
     | BallotLocale
-    | SheetOf<InlineBallotImage>
-    | SheetOf<BallotPageLayout>
+    | SheetOf<InlineBallotImage | null>
+    | SheetOf<BallotPageLayout | null>
```

Comment:
CDF CVRs don't necessarily have images for both sheets.

Reply:
`undefined` is seralized to `null` in an array anyway, so it seemed less confusing to just type it as `null`.